styles/pages/Home.module.scss

```diff
@@ -419,37 +433,44 @@
     flex-direction: column;
     padding: 48px 16px 48px 16px;
     border-radius: 8px;
+    align-items: center;
+    align-content: center;
+
     color: $primary-black-1;
-    background-color: #f5f5f5;
+    background-color: #ffffff;
 
     .statRow {
-      width: 800px;
+      width: 1200px;
+      display: flex;
+      flex-direction: row;
```

Comment:
flex-direction is default row, so we don't need this line

Reply:
Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1